src/App.tsx

```diff
@@ -1,13 +1,25 @@
-
-
-
+import { Link } from "react-router-dom";
+import axios from "axios";
+import { useEffect, useState } from "react";
 export default function App() {
   
-
+  const [randomURL, setRandomURL] = useState("")
```

Comment:
```suggestion
  const [randomURL, setRandomURL] = useState<string>("")
```
This should have a type 'string'